include/core/density_model.h

```diff
@@ -51,16 +41,58 @@ class DensityConstant : public DensityModel
   {}
 
   /**
-   * @brief Returns the denisty
-   *
-   * @param temperature Temperature at time t+dt
+   * @brief value Calculates the value of a physical property.
```

Comment:
Since we are in the density class, shouldn't we say @brief value Calculates the density ?

Reply:
We ab-so-lu-tely should